.cirrus.yml

```diff
@@ -1,133 +1,186 @@
-        GCPPROJECT: "libpod-218412"
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    clone_script: &noop mkdir -p $CIRRUS_WORKING_DIR
-    script: /usr/local/bin/entrypoint.sh
+  # Runs within Cirrus's "community cluster"
+  container:
+    image: "$FEDORA_CONTAINER_FQIN"
+    cpu: 4
+    memory: 12
 
+  timeout_in: 20m
 
-gating_task:
-    name: "Gating test"
-    alias: gating
+  depends_on:
+    - build
+
+  script:
+    - "$SCRIPT_BASE/enable_podman.sh"
+    - make validate
 
-    # Only run this on PRs, never during post-merge testing.  This is also required
-    # for proper setting of EPOCH_TEST_COMMIT value, required by validation tools.
-    only_if: $CIRRUS_PR != ""
+test_task:
+  name: "Test on $FEDORA_NAME"
+  alias: test
 
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    # Runs within Cirrus's "community cluster"
-    container:
-        image: "$FEDORA_CONTAINER_FQIN"
-        cpu: 4
-        memory: 12
+  depends_on:
+    - gating
 
-    timeout_in: 20m
+  script:
+    - "$SCRIPT_BASE/test.sh"
 
-    script:
-        - export PATH="$PATH:$GOPATH/bin"
-        - make
-        - go get github.com/vbatts/git-validation
-        - make validate
+latest_task:
+  name: "Podman latest test"
+  alias: podman_latest
 
-test_task:
-    name: "Test on $FEDORA_NAME"
-    alias: test
+  # Only run this on PRs, never during post-merge testing.  This is also required
+  # for proper setting of EPOCH_TEST_COMMIT value, required by validation tools.
+  only_if: $CIRRUS_PR != ""
+
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+  # Runs within Cirrus's "community cluster"
+  container:
+    image: "$FEDORA_CONTAINER_FQIN"
+    cpu: 4
+    memory: 12
 
-    depends_on:
-        - gating
+  timeout_in: 20m
 
-    script:
-        - "$SCRIPT_BASE/latest_podman.sh"
-        - "$SCRIPT_BASE/test.sh"
+  depends_on:
+    - test
 
+  script:
+    - export PATH="$PATH:$GOPATH/bin"
+    - "$SCRIPT_BASE/latest_podman.sh"
+    - "$SCRIPT_BASE/enable_podman.sh"
+    - "$SCRIPT_BASE/test.sh"
 
 # Status aggregator for all tests.  This task simply ensures a defined
 # set of tasks all passed, and allows confirming that based on the status
 # of this task.
 success_task:
-    name: "Total Success"
-    alias: success
-
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
-
-    # N/B: ALL tasks must be listed here, minus their '_task' suffix.
-    depends_on:
-        - meta
-        - gating
-        - test
-    container: *smallcontainer
-    clone_script: *noop
-    script: *noop
+  name: "Total Success"
+  alias: success
+
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+
+  # N/B: ALL tasks must be listed here, minus their '_task' suffix.
+  depends_on:
+    - meta
+    - gating
```

Comment:
don't forget to add `build`

Reply:
I was assuming the task dependency tree would take care of that for me. Will add it.

Reply:
It will, but that's hard for old human brains to process.  Also, there may be skips/only_if clauses in play, which makes it more hard for old brains.  Easier to just list everything here and rely on Cirrus doing the right thing.

Reply:
[In case it helps, Ed made a visualization tool for this very reason.](https://github.com/containers/automation/tree/main/cirrus-task-map)

Reply:
I can play will with others. :-) All tasks listed.